modules/javafx.graphics/src/main/java/com/sun/javafx/iio/common/ImageTools.java

```diff
@@ -667,8 +667,8 @@ private static void computeUpdatedPixels(int sourceOffset,
                     finalHeight = (int) ((float) sourceHeight * finalWidth / sourceWidth);
```

Comment:
I think the above two blocks should also round instead of truncate. Otherwise the smaller dimension will be calculated differently depending on whether the requested size is 0 or is a larger-than-needed number.

Reply:
Alright, I rounded this as well, and covered these in the unit test.